frontend/app-development/features/policyEditor/SettingsModal/SettingsModal.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import classes from './SettingsModal.module.css';
+import { Button, Heading } from '@digdir/design-system-react';
+import { useTranslation } from 'react-i18next';
+import { CogIcon } from '@navikt/aksel-icons';
+import { Modal } from 'app-shared/components/Modal';
+
+/**
+ * Displays the settings modal. It displays the content inside the Modal component.
+ *
+ * @returns {React.ReactNode}
+ */
```

Comment:
Tenker at det ikke er nødvendig å beskrive impl-detaljer i denne kommentaren. :)
```suggestion
/**
 * Displays the settings modal.
 *
 * @returns {React.ReactNode}
 */
```